components/api_server/src/database/measurements.py

```diff
@@ -27,52 +36,24 @@ def count_measurements(database: Database) -> int:
     return int(database.measurements.estimated_document_count())
 
 
-def measurements_by_metric(
-    database: Database,
-    *metric_uuids: MetricId,
-    min_iso_timestamp: str = "",
-    max_iso_timestamp: str = "",
-):
-    """Return recent measurements for the specified metrics, without entities and issue status."""
-    if not metric_uuids:
-        return []
-    measurement_filter: dict = {"metric_uuid": {"$in": metric_uuids}}
-    if min_iso_timestamp:  # pragma: no feature-test-cover
-        measurement_filter["end"] = {"$gt": min_iso_timestamp}
-    if max_iso_timestamp:  # pragma: no feature-test-cover
-        measurement_filter["start"] = {"$lt": max_iso_timestamp}
-    return list(
-        database.measurements.find(
-            measurement_filter,
-            sort=[("start", pymongo.ASCENDING)],
-            projection={
-                "_id": False,
-                "sources.entities": False,
-                "sources.entity_user_data": False,
-                "issue_status": False,
-            },
-        ),
-    )
+def measurements_in_period(database: Database, min_iso_timestamp: str, max_iso_timestamp: str):
+    """Return recent measurements for the specified metrics, without source details and issue status."""
+    measurement_filter = {"start": {"$lte": max_iso_timestamp or iso_timestamp()}, "end": {"$gte": min_iso_timestamp}}
```

Comment:
I don't get this; start on less-than-or-equals-to MAX, end on greater-than-or-equals to MIN, but sort ascending?
I would expect either the start/end to be reversed, or the sorting to be descending. Perhaps describe/comment?

Reply:
Comment added:

```python
    # Return measurements that partially or completely overlap with the period, meaning they have their start before
    # the end of the period (max_iso_timestamp) and their end after the start of the period (min_iso_timestamp):
```